app/controllers/articles_controller.rb

```diff
@@ -3,22 +3,27 @@ class ArticlesController < ApplicationController
   before_action :set_article, only: [:show, :edit, :update, :destroy]
   after_action :verify_authorized, except: :index
 
+
   def index
+    set_meta_tags title: 'Articles'
```

Comment:
do this after authorizing as the last item on the method. the priority of this is low in all the actions